### PR TITLE
[hue] Improved code (notify status listener)

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueBridgeHandler.java [hunk 1]
```diff
@@ -79,8 +79,24 @@
 @NonNullByDefault
 public class HueBridgeHandler extends ConfigStatusBridgeHandler implements HueClient {
 
-    private long lightPollingInterval = TimeUnit.SECONDS.toSeconds(10);
-    private long sensorPollingInterval = TimeUnit.MILLISECONDS.toMillis(500);
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = Collections.singleton(THING_TYPE_BRIDGE);
+
+    private static final String DEVICE_TYPE = "EclipseSmartHome";
+
+    private static enum STATUS_TYPE {
```
Comment (cpmeister) [inline bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueBridgeHandler.java:86]: Please change enum name to camelcase

Comment (lolodomo) [reply bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueBridgeHandler.java]: You're right. It is fixed.

**Comment by lolodomo** [comment]
If you could merge this one, I will then update my PR about the hue groups because this will introduce a confliuct.

File: bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueBridgeHandler.java [hunk 2]
```diff
@@ -79,8 +79,24 @@
 @NonNullByDefault
 public class HueBridgeHandler extends ConfigStatusBridgeHandler implements HueClient {
 
-    private long lightPollingInterval = TimeUnit.SECONDS.toSeconds(10);
-    private long sensorPollingInterval = TimeUnit.MILLISECONDS.toMillis(500);
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = Collections.singleton(THING_TYPE_BRIDGE);
+
+    private static final String DEVICE_TYPE = "EclipseSmartHome";
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueBridgeHandler.java:84]: I think it is time to change this to "openHAB".

**Pull request merged**